dashboard/app/models/lesson.rb

```diff
@@ -279,6 +279,9 @@ def summarize(include_bonus_levels = false, for_edit: false)
       if has_lesson_plan
         lesson_data[:lesson_plan_html_url] = lesson_plan_html_url
         lesson_data[:lesson_plan_pdf_url] = lesson_plan_pdf_url
+        if script.include_student_lesson_plans && script.is_migrated
+          lesson_data[:student_lesson_plan_html_url] = script_lesson_path(script, self) + '/student'
```

Comment:
I think you can use `script_lesson_student_path` here

![Screen Shot 2021-03-04 at 2 01 54 PM](https://user-images.githubusercontent.com/46464143/110036664-3353ec80-7cf2-11eb-867a-7eef4b94f358.png)

Reply:
Sorry for the bad screenshot 😬

Reply:
Oh! I went looking to see if I could find this before! Nice! There is a spot in the previous PR that could benefit from this too

Reply:
Fixed! I'll go back into the previous PR and look for that spot to change